lib/perl/Genome/Qc/Config.pm

```diff
@@ -27,6 +27,7 @@ sub get_commands_for_alignment_result {
                 use_version => 1.123,
                 output_file=> 'output_file',
                 chart_output => 'chart_output',
+                # metric_accumulation_level => ['SAMPLE'], #not supported in 1.123
             },
         },
         picard_mark_duplicates => {
```

Comment:
We should remove MarkDuplicates as well.  The alignment strategy should define the duplicate marking.  For CCDG SAMBLASTER is run as part of SpeedSeq realign.  The collection of duplicate reads to calculate duplication rate can be achieved with samtools flagstat.